types/pollyjs__node-server/index.d.ts

```diff
@@ -0,0 +1,43 @@
+// Type definitions for @pollyjs/node-server 2.6
+// Project: https://github.com/netflix/pollyjs/tree/master/packages/@pollyjs/node-server
+// Definitions by: Offir Golan <https://github.com/offirgolan>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.5
```

Comment:
This should be the minimum version of TypeScript compatible with these type definitions - TS 3.5 seems high, but I might be missing something.

Reply:
Yeah, that was my bad. Reduced it to 2.4.